samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/doc/ModelEnumClass.md

```diff
@@ -1,4 +1,4 @@
-# openapi.model.EnumClass
+# openapi.model.ModelEnumClass
```

Comment:
I think this needs to be noted as a breaking change

Reply:
This is only happening in dart-dio and anything with that name would not have compiled up until now. Since `EnumClass` is a name coming from the spec.